changelogs/unreleased/4382-jxun

```diff
@@ -0,0 +1 @@
+feat: migrate velero controller from kubebuilder v2 to v3
```

Comment:
This changelog should include details to state that v1beta1 CRDs are no longer supported and that we only support Kubernetes 1.16 and later.

Reply:
Got it. Thanks.